nautobot_golden_config/forms.py

```diff
@@ -462,46 +463,30 @@ class Meta:
 # ConfigPlan
 
 
-class ConfigPlanFilterForm(NautobotFilterForm):
-    """Filter Form for ConfigPlan."""
+class ConfigPlanForm(NautobotModelForm):
+    """Form for ConfigPlan instances."""
 
-    model = models.ConfigPlan
+    plan_type = forms.ChoiceField(choices=add_blank_choice(ConfigPlanTypeChoice), required=False, label="Plan Type")
```

Comment:
```suggestion
    plan_type = forms.ChoiceField(choices=add_blank_choice(ConfigPlanTypeChoice), required=True, label="Plan Type")
```

All plans should have a type.